paimon-filesystems/paimon-s3-impl/src/main/resources/META-INF/NOTICE

```diff
@@ -14,7 +14,6 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.amazonaws:jmespath-java:1.12.319
 - com.fasterxml.jackson.core:jackson-annotations:2.14.2
 - com.fasterxml.jackson.core:jackson-core:2.14.2
-- com.fasterxml.jackson.core:jackson-databind:2.14.2
```

Comment:
Why update here?

Reply:
I made changes based on UT.

https://github.com/apache/incubator-paimon/actions/runs/8228264920/job/22497412307#step:5:80

<img width="1058" alt="image" src="https://github.com/apache/incubator-paimon/assets/37063904/19253431-c4e7-4e53-9360-f5f3142e31e1">

Reply:
Sorry, the notice checking UT is flawed, I'll check it later.

Reply:
I have restored this file.